client/src/components/ListingEntry.jsx

```diff
@@ -51,12 +51,12 @@ class ListingEntry extends React.Component {
               <ReviewCount>{`(${this.props.listing.reviewCount})`}</ReviewCount>
             </Rating>
             <RoomDescription>
-              <Text>{`${this.props.listing.roomType} · `}</Text>
+              <Text>{`${this.props.listing.room_type} · `}</Text>
```

Comment:
Is room here referring to the same idea as property? If so, might make maintenance easier to use the word "property" throughout the entire codebase. Or am I misunderstanding what room refers to here?

Reply:
same as above answer